### PR TITLE
Chore/cherry pick 16 changes

### PR DESCRIPTION
**Pull request by makelicious**

### CODE REVIEW

**Comment by Zangetsu101** [comment]
@makelicious this is ok to merge right?

**Comment by Zangetsu101** [comment]
Also these changes need to go to countryconfig :sweat_smile:

**Comment by rikukissa** [comment]
@makelicious @Zangetsu101 closing this for now. Please reopen if this was still going to merged

**Comment by Zangetsu101** [comment]
@makelicious This does need to be merged right? The current qa is actually running this branch I think

File: CHANGELOG.md
Comment (Zangetsu101) [inline CHANGELOG.md]: This shouldn't be here. It should be merged with line 52 instead

**Comment by makelicious** [comment]
@Zangetsu101 these are already in countryconfig, too. I deployed this to qa last week, since the environment was broken. Depending on the process we have for updating `release-v1.6.0` branch, we can merge this or if you have some preferred way of doing it

**Comment by Zangetsu101** [comment]
I have just noticed a anomaly in the CHANGELOG. We can go ahead and merge this in after it's fixed

**Comment by makelicious** [comment]
@Zangetsu101 Thanks! Did I interpret it correctly that the extra title was the problem? Comparing to countryconfig the changes were actually related to 1.5

**Comment by Zangetsu101** [comment]
> @Zangetsu101 Thanks! Did I interpret it correctly that the extra title was the problem? Comparing to countryconfig the changes were actually related to 1.5

Yes that's the one

File: CHANGELOG.md
Comment (Zangetsu101) [inline CHANGELOG.md]:
```suggestion
## [1.4.1](https://github.com/opencrvs/opencrvs-farajaland/compare/v1.4.0...v1.4.1)
```

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**
